README.md

```diff
@@ -19,3 +19,8 @@ Enable Docker Extensions
 <img width="1377" alt="image" src="https://user-images.githubusercontent.com/18168330/194485464-ab8e402f-8e07-4a0c-86e5-3dae39623df8.png">
 
 <img width="1376" alt="image" src="https://user-images.githubusercontent.com/18168330/194485514-4991ccee-15a4-4df4-b407-c57a2a77d372.png">
+
+Setting up a new developer machine can be time consuming and tricky. The given steps for local development helps to simplify the process with easy-to-understand instructions as follows:
```

Comment:
can you put the exact steps from here (https://www.docker.com/blog/build-your-first-docker-extension/) under subsections like

## Development

### Enable hot reloading
```
cd ui
npm start
docker extension dev ui-source deepfenceio/yarahunter-extension http://localhost:3000
```

### Open Chrome Dev Tools
In order to open the Chrome Dev Tools for your extension when clicking on the extension tab, run:

`docker extension dev deepfenceio/yarahunter-extension`

Each subsequent click on the extension tab will also open Chrome Dev Tools. To stop this behavior, run:

`docker extension dev reset deepfenceio/yarahunter-extension`

### Change the backend service code
```
make build-extension
make update-extension
```

Reply:
and at then end put reference: https://www.docker.com/blog/build-your-first-docker-extension/